app/src/main/java/pl/futurecollarc/invoicing/db/files/FileDatabase.java

```diff
@@ -0,0 +1,75 @@
+package pl.futurecollarc.invoicing.db.files;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import pl.futurecollarc.invoicing.config.Config;
+import pl.futurecollarc.invoicing.db.Database;
+import pl.futurecollarc.invoicing.model.Invoice;
+import pl.futurecollarc.invoicing.service.FilesService;
+import pl.futurecollarc.invoicing.service.JsonService;
+
+@Data
+@AllArgsConstructor
+public class FileDatabase implements Database {
+
+  private final List<String> allInvoices = new ArrayList<>();
```

Comment:
Dalej masz to pole, które burzy cały sens działania tej bazy. Jeżeli potrzebujesz aktualnej listy invoice, powinnaś ją sobie za każdym razem przeczytać z pliku, a nie brać z pamięci :)